nemo/collections/asr/parts/conformer_modules.py

```diff
@@ -124,7 +125,7 @@ class ConformerConvolution(nn.Module):
         kernel_size (int): kernel size for depthwise convolution
     """
 
-    def __init__(self, d_model, kernel_size):
+    def __init__(self, d_model, kernel_size, norm_type="batch_norm"):
```

Comment:
Same comment as above

Reply:
dropped the norm_type as it was experimental.